content/akeneo-event-platform/getting-started.md

```diff
@@ -22,33 +22,32 @@ This guide provides a step-by-step walkthrough to help you set up event subscrip
 In this example, we will create a new `connection` in the PIM and use it to generate an `API token`.
 :::
 
-1. **Create a connection in Akeneo PIM:**
+**1. Create a connection in Akeneo PIM:**
    - Navigate to **Connect** > **Connection settings** > **Create**.
    - Fill out the form to create the connection.
    - Note the generated `Client ID`, `Secret`, `Username`, and `Password`.
 
-2. **Set Your Environment Variables:**
+**2. Set Your Environment Variables:**
    - Define the Client ID, Secret, Username, Password, and Akeneo host URL as environment variables:
-
```

Comment:
I think this line break was used to avoid the weird indent that you can see in the PR env

Reply:
hum I'll take a look thanks

Reply:
fixed, issue was the indent on the bash line